docs/src/prototypes/chatMessages/ControlMessages/GroupControlMessages.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import * as keyboardKey from 'keyboard-key'
+
+import { List, chatBehavior, Accessibility, Flex, Icon } from '@stardust-ui/react'
+import ControlMessage from './ControlMessage'
+
+const controlMessagesGroupBehavior: Accessibility<any> = props => {
+  const behaviorData = chatBehavior(props)
+
+  behaviorData.attributes.root = {
+    ...behaviorData.attributes.root,
+    'data-is-focusable': true,
+  }
+
+  behaviorData.focusZone = {
+    mode: behaviorData.focusZone.mode,
+    props: {
+      ...behaviorData.focusZone.props,
+      shouldFocusOnMount: true,
+      shouldFocusInnerElementWhenReceivedFocus: true,
+      defaultTabbableElement: undefined,
+    },
+  }
+  return behaviorData
+}
+
+const GroupControlMessages = () => {
+  const [expanded, setExpanded] = React.useState(false)
+  const [focused, setFocused] = React.useState(false)
+
+  const renderItems = () => {
+    return _.map(groupControlMessageItems, item => {
+      return {
+        content: <ControlMessage message={item} />,
+        styles: { padding: 0, display: 'block', minHeight: '25px' },
+      }
+    })
+  }
+
+  return (
+    <Flex
+      onKeyDown={e => {
+        const eventCode = keyboardKey.getCode(e)
+        if (eventCode === keyboardKey.Enter) {
```

Comment:
Should we handle Space here as well?

Reply:
Only `Enter`, based on current behavior in WebClient